tile_generator/config.py

```diff
@@ -61,6 +64,12 @@
 # Normalize Jobs - Ensure that job type, template, and properties are set for
 # every job
 
+def _find_git_root():
+	try:
+		git_root = subprocess.check_output(["git", "rev-parse", "--show-toplevel"]).decode("utf-8").strip()
```

Comment:
I always prefer a library call to a subprocess and would recommend using gitpython here: some docs FYI: https://gitpython.readthedocs.io/en/stable/quickstart.html#display-level-1-contents

Also the working_dir may be the same as the git_root expected here, I'm not certain - https://gitpython.readthedocs.io/en/stable/reference.html#git.cmd.Git.working_dir

Reply:
The reason I opted for this is because I had to write a file system walk to locate the root of the git repo when this was originally gitpython. As in, if the tile isn't in the repo root then I'd need to walk up the parent dirs. I'll take a look at both of these. 

The standard in other parts of the code for both `bosh` and `git` is to use subprocess calls:  https://github.com/cf-platform-eng/tile-generator/blob/abeca18e1f86d3d09d6f6e4f9d154c8426407092/tile_generator/opsmgr.py#L69

```python
def get_credential_dir(update=False):
	dir = find_credential_dir()
	if update:
		devnull = open(os.devnull,"w")
		subprocess.call(['git', 'pull'], cwd=dir, stdout=devnull, stderr=devnull)
	return dir
```

I think for these reasons I'm inclined to still use subprocess unless `working_dir` pans out (and anywhere else I was adding `git` usage). Happy to settle on gitpython if you'd prefer. Third option: a follow up to refactor all the subprocess calls for git so we're consistently either using gitpython or subprocess, but not both.

Reply:
A subsequent refactor to maintain consistency would get my vote.